sigilrecorder.lic

```diff
@@ -0,0 +1,205 @@
+custom_require.call(%w[common common-arcana common-items common-travel drinfomon events spellmonitor equipmanager])
+
+class SigilRecorder
+  include DRC
+  include DRCA
+  include DRCI
+  include DRCT
+
+	def initialize
+		require 'yaml'
+		@data = YAML.load(File.read("./scripts/data/base-sigils.yaml"))
+		@city_list = ['shard', 'crossing', 'riverhaven', 'other']
+		@sigil_list = ['abolition','congruence','induction','permutation','rarefaction', 'antipode','ascension','clarification','decay','evolution','integration','metamorphosis','nurture','paradox','unity']
+		@settings = get_settings
+		get_region
+		get_season #make sure we're recording sigils to the correct season
+		region_room_list #generate the walk_list array
+		generate_nogo_list
+		
+		Flags.add('danger-room', /.*begins to advance on you.*|You weave back and forth in a dance-like motion|You must be closer to use tactical abilities on your opponent/)
+		@walk_list.each do |roomid|
+			@data['nogo'].include?(roomid) ? next : (travel_result = walk_to(roomid, false))
+			respond("Travel Debug: #{travel_result} | Room Target: #{roomid}")
+			if Room.current.id == nil
```

Comment:
Is `roomid` always the current room? What if travel failed (e.g. `travel_result` is false)?